### ISSUE TITLE
source code for training of senseBert

### ISSUE THREAD

**Issue by papasanimohansrinivas** [AI21Labs/sense-bert#1]
guys are u planning to release sensebert training code and dataset , if so when @yoel-zeldes @oriram

**Comment by oriram** [comment]
Hi @papasanimohansrinivas , thanks for your interest! 
Unfortunately, we'll not release the training code, as it's proprietary.
Basically the code is straightforward - we add another head, parallel to BERT's Masked LM head.

The training data is exactly the same as BERT's (Wiki and BookCorpus), with the exception that we add supersense labels to masked words (according to WordNet), as described in the paper.
